packages/services/src/epcisEvents/types.ts

```diff
@@ -42,7 +42,7 @@ export interface IContext {
   vckit: IVCKitContext;
   dlr: IConfigDLR;
   storage: StorageServiceConfig;
-  identifierKeyPath: string;
+  identifierKeyPath: string | any;
```

Comment:
Hi, what does identifierKeyPath look like when it is `any`?

Reply:
The identifierKeyPath now can be `string` for the path like /productIdentifier/0/identifierValue or object to construct the appropriate identifier like
```
{
...
  "identifierKeyPath": {
    "function": "concatService",
    "args": [
      { "type": "text", "value": "(01)" },
      { "type": "path", "value": "/productIdentifier/0/identifierValue" },
      { "type": "text", "value": "(10)" },
      { "type": "path", "value": "/batchIdentifier/0/identifierValue" },
      { "type": "text", "value": "(21)" },
      { "type": "path", "value": "/itemIdentifier/0/identifierValue" }
    ]
  }
...
}
```

Reply:
There is no need to update now, but should this be `Record<string, string>` in the future?